3d_segmentation/unetr_segmentation_3d.ipynb

```diff
@@ -0,0 +1,4317 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    img_name = val_ds[case_num]['image_meta_dict']['filename_or_obj'].split('/')[-1]
</pre><p>Suggest to use <code>os.path.split()</code>  to support non-linux path separators.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/263/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Thanks @Nic-Ma. Fixed.